cardano-node-chairman/test/Test/Cardano/Node/Chairman.hs

```diff
@@ -30,12 +32,13 @@ import qualified System.Process as IO
 {- HLINT ignore "Redundant <&>" -}
 
 prop_spawnOneNode :: Property
-prop_spawnOneNode = H.propertyOnce . H.workspace "temp/chairman" $ \tempDir -> do
+prop_spawnOneNode = H.propertyOnce . H.workspace "x" $ \tempDir -> do
```

Comment:
To avoid causing `socketDir` being assigned a string of length exceeding `104`, which prevents it from being a valid Unix Domain Socket name.